packages/kit/src/exports/vite/index.js

```diff
@@ -561,7 +561,7 @@ function kit() {
 
 function check_vite_version() {
 	// TODO parse from kit peer deps and maybe do a full semver compare if we ever require feature releases a min
-	const min_required_vite_major = 3;
+	const min_required_vite_major = 4;
```

Comment:
is this function a good place to link to this PR so people get to proper migration instructions faster?

Reply:
don't think it's necessary as all they need to do is bump the version number